src/Uno.UI.FluentTheme.v2/Resources/Version2/PriorityDefault/AppBarButton_themeresources.xaml

```diff
@@ -107,15 +108,15 @@
     <Thickness x:Key="AppBarButtonInnerBorderCompactMargin">2,6,2,22</Thickness>
     <Thickness x:Key="AppBarButtonInnerBorderOverflowMargin">4,0,4,0</Thickness>
 
-    <Style TargetType="AppBarButton" BasedOn="{StaticResource DefaultAppBarButtonStyle}" />
+    <Style TargetType="local:AppBarButton" BasedOn="{StaticResource DefaultAppBarButtonStyle}" />
```

Comment:
Similar question here for `AppBar`, `AppBarButton`, etc. These are not recommended for use in WinUI, so having them in `Windows.UI.Xaml.Controls` in "UWP mode" would make more sense to me.